dev/com.ibm.ws.security.utility_fat_one/fat/src/com/ibm/ws/security/utility/test/SecurityUtilityGenerateAesKeyTest.java

```diff
@@ -0,0 +1,567 @@
+        Matcher matcher = ksPattern.matcher(v1Output.getStdout());
+        assertTrue("Could not find keyStore snippet in stdout", matcher.find());
+        String keystoreSnippet = matcher.group();
+
+        String passwordsnippet = getPasswordOverride(keystoreSnippet,ltpaSnippet, aesConfigFile);
+        writeStringToServerOverride(passwordsnippet, testServer);
+
+
+         // setup
+        testServer.startServer(testName.getMethodName());
+        
+        // Verify startup log contains LTPA initialization
+        assertNotNull("Expected LTPA configuration ready message not found in the log.",
+                      testServer.waitForStringInLogUsingMark("CWWKS4105I", 5000));
+
+        // Verify startup log contains the keystore loaded successfully message
+        assertNotNull("Expected Keystore loaded message not found in the log.",
+                      testServer.waitForStringInLogUsingMark("Successfully loaded default keystore", 5000));
+
+
+        testServer.stopServer();
+    }
+
+    /**
+     * Test that verifies the generated XML file is properly included in the Liberty server
+     * configuration using the directions from the command success output.
+     */
+    @Test
+    public void testGeneratedXmlFileInclusion() throws Exception {
+        testEnvironment = new Properties();
+        testEnvironment.put("JVM_ARGS", "-Dcom.ibm.ws.beta.edition=true");
+
+        // Generate encryption XML file
+        String includeFileName = "include.xml";
+        String xmlFilePath = testOutputDir + "/" + includeFileName;
+        File xmlFile = new File(xmlFilePath);
+
+        // Paths
+        File baseServerXml = new File(testServer.getInstallRoot()+"/servers/SecurityUtilityGenerateServer/server.xml");
+        
+
+        ProgramOutput generateCommandOutput = testMachine.execute(
+            securityUtilityPath,
+            new String[] { "generateAESKey", "--createConfigFile=" + xmlFilePath},
+            libertyInstallRoot,
+            testEnvironment);
+
+        Log.info(thisClass, testName.getMethodName(), "Generate command - stderr:\n" + generateCommandOutput.getStderr());
+        Log.info(thisClass, testName.getMethodName(), "Generate command - stdout:\n" + generateCommandOutput.getStdout());
+        Log.info(thisClass, testName.getMethodName(), "Generate command - Return code: " + generateCommandOutput.getReturnCode());
+
+        // Verify success
+        assertEquals("generateAESKey command should succeed", SUCCESS_RC, generateCommandOutput.getReturnCode());
+        assertTrue("Output file should be created", xmlFile.exists());
+
+        // Verify output instructions
+        String generateOutput = generateCommandOutput.getStdout();
+        assertTrue("Output should mention include directive",
+                  generateOutput.contains("<include location="));
+
+
+        String includesnippet = getincludeOverride(xmlFile.getAbsolutePath());
```

Comment:
instead of creating our own include, the output from generateAESKey should provide the xml needed for this. We should parse the include from the command output and insert that directly into server.xml. This would test that the output from the command is valid.